components/create-collective/CreateCollectiveForm.js

```diff
@@ -210,6 +213,25 @@ class CreateCollectiveForm extends React.Component {
               {formik => {
                 const { values, handleSubmit, errors, touched } = formik;
 
+                const suggestedSlug = () => {
+                  const slugOptions = {
+                    replacement: '-',
+                    lower: true,
+                    strict: true,
+                  };
+
+                  return slugify(values.name, slugOptions);
+                };
+
+                if (!this.state.isURLFieldTouched) {
+                  values.slug = suggestedSlug();
+                }
+
+                const handleSlugChange = e => {
+                  values.slug = e.target.value;
```

Comment:
`values` object is not meant to me modified directly. It works here because we're doing a `setState` just after, but otherwise the component would not update. It's dangerous to have this change, because it may break in the future if Formik change the way they handle their internal state.

There are multiple way to handle changes with Formik, one of the simplest is to use the `setFieldValue` helper (see https://jaredpalmer.com/formik/docs/api/formik). Note that you also have a `isTouched` prop (and associated helpers) that may help you with that code.